libraries/botframework-schema/etc/botframework-schema.api.md

```diff
@@ -953,11 +953,55 @@ export interface MeetingInfo {
     organizer: TeamsChannelAccount;
 }
 
+// @public
+export type MeetingNotification = TargetedMeetingNotification;
+
+// @public
+export interface MeetingNotificationBase<T> {
+    type: string;
+    value: T;
+}
+
+// @public
+export interface MeetingNotificationChannelData {
+    onBehalfOf?: MeetingOnBehalfOf[];
+}
+
+// @public
+export interface MeetingNotificationRecipientFailureInfo {
+    errorCode: string;
+    failureReason: string;
+    recipientMri: string;
+}
+
+// @public
+export interface MeetingNotificationRecipientFailureInfos {
+    recipientsFailureInfo: MeetingNotificationRecipientFailureInfo[];
+}
+
+// @public
+export interface MeetingOnBehalfOf {
+    displayName?: string;
+    itemid: 0 | number;
+    mentionType: 'person' | string;
+    mri: string;
+}
+
+// @public (undocumented)
+export interface MeetingStageSurface<T> {
+    content: T;
+    contentType: 'task' | string;
+    surface: 'meetingStage';
+}
+
 // @public
 export interface MeetingStartEventDetails extends MeetingEventDetails {
     startTime: Date;
 }
 
+// @public
+export type MeetingSurface = MeetingStageSurface<any>;
```

Comment:
This makes me slightly confused. So the list of surfaces should be able to contain MeetingStageSurface<T> and any other surface which is not meeting stage in the future (from the extensibility perspective). If MeetingSurface is set to be meeting stage surface with different content type, what will happen when there is a completely different surface type?

Reply:
Let's say in the future, we have a different meeting surface altogether called `MeetingTabSurface`. We can simply extend `MeetingSurface = MeetingStageSurface<any> | MeetingTabSurface`. And so the list of surfaces can look like the following `surfaces: [meetingStageSurface, meetingTabSurface]` where the `meetingStageSurface` is an object that conforms to the `MeetingStageSurface<any>` type, and `meetingTabSurface` conforms to type `MeetingTabSurface`. And that `surfaces` will conform to `MeetingSurface[]` type. The pipe `|` symbol stands for OR.